RetailCoder.VBE/UI/DockablePresenterBase.cs

```diff
@@ -28,13 +32,18 @@ private Window CreateToolWindow(IDockableUserControl control)
             Window toolWindow;
             try
             {
+                _logger.Trace("Loading \"{0}\" ClassId {1}", control.Caption, control.ClassId);
                 toolWindow = _vbe.Windows.CreateToolWindow(_addin, _DockableWindowHost.RegisteredProgId,
                     control.Caption, control.ClassId, ref userControlObject);
             }
-            catch (COMException)
+            catch (COMException exception)
             {
-                toolWindow = _vbe.Windows.CreateToolWindow(_addin, _DockableWindowHost.RegisteredProgId,
-                    control.Caption, control.ClassId, ref userControlObject);
+                var logEvent = new LogEventInfo(LogLevel.Error, _logger.Name,  "Error Creating Control");
+                logEvent.Exception = exception;
+                logEvent.Properties.Add("EventID", 1);
+
+                _logger.Error(logEvent);
+                throw;
```

Comment:
I don't mind the logging, but I'm not okay with removing the retry until we understand the problem and actually fix it.